### PR TITLE
Update qoqo 1.4

### PR DESCRIPTION
**Pull request by gsilviHQS**
General cargo update + qoqo 1.4

### CODE REVIEW

File: qoqo_aqt/docs/requirements.txt
Comment (kbarkhqs) [inline qoqo_aqt/docs/requirements.txt:1]: Could this file be combined with the pyproject.toml?

Comment (gsilviHQS) [reply qoqo_aqt/docs/requirements.txt:1]: tell me how pls

Comment (gsilviHQS) [reply qoqo_aqt/docs/requirements.txt:1]: done, now libs from requirement.txt are also in pyproject.toml as optional dependencies.
When the reusable workflow update to this convention will go online, requirements.txt can be erased

**Review by kbarkhqs — APPROVED** [review]

File: qoqo_aqt/docs/requirements.txt
Comment (gsilviHQS) [reply qoqo_aqt/docs/requirements.txt:1]: ok, now requirements.txt has been obliterated.

**Pull request merged**